Assets/Scripts/GameManager.cs

```diff
@@ -0,0 +1,14 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameManager : MonoBehaviour
+{
+
+    public GameObject completeLevelUI;
+
+    public void CompleteLevel()
```

Comment:
Nice, I like having a centralised location to handle all game state logic. Nice work!